notebooks/00_quick_start/dkn_synthetic.ipynb

```diff
@@ -22,13 +22,20 @@
     "- DKN uses an attention module to dynamically calculate a user's aggregated historical representaition.\n",
```

Comment:
<p>Do you need me to upload these files to azure storage?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
uploaded